### PR TITLE
add policy-machineconfig-chrony.yaml

### PR DESCRIPTION
**Pull request by ch-stark**
tested policy and checked with author of blog (mentioned in readme)

### CODE REVIEW

File: community/CM-Configuration-Management/policy-machineconfig-chrony.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: add-chrony
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-machineconfig-chrony.yaml]: remove hardcoded namespace

File: community/CM-Configuration-Management/policy-machineconfig-chrony.yaml [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+subjects:
+- name: add-chrony
+  kind: Policy
+  apiGroup: policy.open-cluster-management.io
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+  name: placement-add-chrony
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-machineconfig-chrony.yaml]: remove hardcoded namespace

File: community/CM-Configuration-Management/policy-machineconfig-chrony.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                            data:,server%200.fedora.pool.ntp.org%0A%0Aserver%201.fedora.pool.ntp.org%0A%0Aserver%202.fedora.pool.ntp.org%0A%0Adriftfile%20/var/lib/chrony/drift%0A%0Amakestep%201.0%203%0A%0Artcsync%0A%0Akeyfile%20/etc/chrony.keys%0A%0Aleapsectz%20right/UTC%0A%0Alogdir%20/var/log/chrony%0A
+          remediationAction: enforce
+          severity: low
+  remediationAction: enforce
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+  name: binding-add-chrony
+  namespace: default
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-machineconfig-chrony.yaml]: remove hardcoded namespace

File: community/README.md [hunk 4]
```diff
@@ -45,6 +45,9 @@ Policies in this folder are organized by [NIST Special Publication 800-53](https
     <td><a href="./CM-Configuration-Management/policy-gatekeeper-sample.yaml">Gatekeeper Sample policy</a>: Use the Gate Sample policy to view an example of how a gatekeeper policy can be applied to a managed cluster</td>
     <td>See the <a href="https://github.com/open-policy-agent/gatekeeper">Gatekeeper</a>. **Note**: Gatekeeper controllers must be installed to use the gatekeeper policy.</td>
   </tr>
+  <tr>
+    <td><a href="./CM-Configuration-Management/policy-machineconfig-chrony.yaml">MachineConfig Chrony Sample </a>: Based on this <a href="https://jaosorior.dev/2019/modifying-node-configurations-in-openshift-4.x/"> blog and example </a></td>
+    </tr>
```
Comment (ycao56) [inline community/README.md:51]: missing <td> and table view is not complete 
image
Can you add something in this table field? I think you can add something like `requires managed cluster to be openshift`

File: community/README.md [hunk 5]
```diff
@@ -45,6 +45,9 @@ Policies in this folder are organized by [NIST Special Publication 800-53](https
     <td><a href="./CM-Configuration-Management/policy-gatekeeper-sample.yaml">Gatekeeper Sample policy</a>: Use the Gate Sample policy to view an example of how a gatekeeper policy can be applied to a managed cluster</td>
     <td>See the <a href="https://github.com/open-policy-agent/gatekeeper">Gatekeeper</a>. **Note**: Gatekeeper controllers must be installed to use the gatekeeper policy.</td>
   </tr>
+  <tr>
+    <td><a href="./CM-Configuration-Management/policy-machineconfig-chrony.yaml">MachineConfig Chrony Sample </a>: Based on this <a href="https://jaosorior.dev/2019/modifying-node-configurations-in-openshift-4.x/"> blog and example </a></td>
```
Comment (ycao56) [inline community/README.md]: can you add a short description of what this policy does?

**Review by dockerymick — APPROVED** [review]
Made a couple of updates. It looks good

File: community/CM-Configuration-Management/policy-machineconfig-chrony.yaml [hunk 6]
```diff
@@ -0,0 +1,71 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: add-chrony
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: PR.IP Information Protection Processes and Procedures, PR.PT Protective Technology
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-machineconfig-chrony.yaml]: categories and controls are not correct

**Comment by dockerymick** [comment]
@ch-stark some of my suggested edits were removed. I will update it now

**Comment by dockerymick** [comment]
Updates are complete @ch-stark

**Review by ycao56 — APPROVED** [review]

**Pull request merged**